nautobot/dcim/tests/test_views.py

```diff
@@ -161,7 +161,7 @@ def create_test_device(name):
     return device
 
 
-class LocationTypeTestCase(ViewTestCases.OrganizationalObjectViewTestCase):
+class LocationTypeTestCase(ViewTestCases.PrimaryObjectViewTestCase):
```

Comment:
This is creating a slight misalignment since LocationType is technically an OrganizationModel. Should we do this instead?
```suggestion
class LocationTypeTestCase(ViewTestCases.OrganizationalObjectViewTestCase, BulkEditObjectsViewTestCase):
```

Reply:
That's what I thought too. updated